keyboards/anavi/knob1/glcdfont.c

```diff
@@ -0,0 +1,232 @@
+#include "progmem.h"
```

Comment:
This is just the corne font file. This file should be removed, as the default font file should be fine.

Reply:
Thank you for the code review. Done. I've forced pushed the modified commit. Thanks, Leon